### DISCUSSION TITLE
How to enable submit button in onBlur mode?

### DISCUSSION THREAD

**Discussion by ArvindNikam76** [react-hook-form/react-hook-form#1776] [Q&A]
See below link for more
https://codesandbox.io/s/react-hook-form-validation-onblur-xtimf

formState.isValid updating late

I'm using mode onBlur but button is still disable even if i filled all field, it requires me to move from last field then is enable

**Comment by bluebill1049** [comment]
you will use `triggerValidation` for specific field, if you want to validate onChange

**Comment by ArvindNikam76** [comment]
then it will start printing error that i don't want initially,

< button disabled={Object.keys(errors).length > 0 || (formState.dirtyFields.size < 5 && !formState.isValid)} >

i'm trying this solution

**Reply by ArvindNikam76** [reply]
reValidateMode:"onChange" should update formState.isValid state on changing. but itis not

**Reply by bluebill1049** [reply]
codesandbox?

**Comment by ArvindNikam76** [comment]
can we trigger validation without printing errors?

**Reply by bluebill1049** [reply]
there is `formState` for you to show and hide error as well, eg dirtyField touched
